### DISCUSSION TITLE
Interested in help with the wiki translation to spanish

### DISCUSSION THREAD

**Discussion by jhonedisonrod21** [linuxdeepin/developer-center#6796] [Features Request & Ideas | 特性请求 & 头脑风暴]
I'm an enthusiast on this beautiful distribution, and I'm interested in the translation of the wiki to Spanish. Is there any way that I can contribute in that matter?

**Comment by BLumia** [comment]
Yeah, you can! There is a translation related Special Interest Group in deepin community. You can checkout [This link](https://www.deepin.org/index/docs/sig/sig/deepin-translation-program/README.md) to get in touch with this group, and feel free to let us know if you need any help while going through this process :)
